modules/session/mod_session_dbd.c

```diff
@@ -571,6 +571,9 @@ static const char *set_cookie_name(cmd_parms * cmd, void *config, const char *ar
     char *line = apr_pstrdup(cmd->pool, args);
     session_dbd_dir_conf *conf = (session_dbd_dir_conf *) config;
     char *cookie = apr_strtok(line, " \t", &last);
+    if(!cookie) {
+        return "Invalid directive, skipping";
```

Comment:
The string has "skipping" but won't this result in a startup error?  Maybe "SessionDBDCookieName requires at least one argument"?